intellij-plugin-structure/structure-dotnet/src/main/kotlin/com/jetbrains/plugin/structure/dotnet/version/Version.kt

```diff
@@ -65,4 +67,42 @@ data class ReSharperVersion(val components: List<Int>, override val productCode:
     else {
       this
     }
+
+  fun getLowerVersion(): ReSharperVersion {
+    if (components.size >= 3 && components[2] != 0) {
+      val minorValue = components[2]
+      require(minorValue > 0) {
```

Comment:
Could you please elaborate on this check? How this can be possible that any of components is lower than 0? Could you please write a test for this scenario?

Reply:
Yeah, it seems to be redundant, it's strange for them to be lower than 0, thanks. will fix.